src/granite/query/executors/value.cr

```diff
@@ -14,7 +14,9 @@ module Granite::Query::Executor
 
       Model.adapter.open do |db|
         db.query_one? @sql, @args do |record_set|
-          return record_set.read.as Scalar
+          val = record_set.read
+          return val.to_f if val.is_a?(PG::Numeric)
```

Comment:
unfortunately, `PG::Numeric` will not exist if you don't include the pg driver.

Reply:
:confused: Ah man...

Reply:
There are ways around that. Granite can define a bare class so the const is around. [mosquito does that with granite::base](https://github.com/robacarp/mosquito/blob/master/src/external_classes.cr) and it's worked well.

Reply:
That could work.  I'll have to revisit this and try that.